src/users/users.controller.ts

```diff
@@ -57,7 +57,8 @@ export class UsersController {
   async join(@Body() content: JoinRequestDto) {
     const { email, password } = content;
     // console.log(email, ' : test이메일 출력');
-    return await this.usersService.createUser(email, password);
+    const user = await this.usersService.createUser(email, password);
+    return UsersDto.fromEntity(await user);
```

Comment:
```suggestion
    const user = await this.usersService.createUser(email, password);
    return UsersDto.fromEntity(user);
```

await은 Promise를 Blocking 방식으로 기다릴 때 사용하는 거에요. 이미 60번 라인에서 기다려서 값을 받아왔는데, 또 쓸 필요는 없죠. 지워주세요.